client/src/js/components/bhFluxSelect.js

```diff
@@ -0,0 +1,50 @@
+angular.module('bhima.components')
+  .component('bhFluxSelect', {
+    templateUrl : 'modules/templates/bhFluxSelect.tmpl.html',
+    controller  : FluxSelectController,
+    bindings    : {
+      onChange : '&',
+      fluxId : '<?',
```

Comment:
It looks like this is a `ui-select multiple`.  Should this name be `fluxIds` then (in plural)?  Since it could take multiple flux ids?